src/Generator.cpp

```diff
@@ -1608,7 +1617,7 @@ void GeneratorInputBase::set_def_min_max() {
 }
 
 Parameter GeneratorInputBase::parameter() const {
-    internal_assert(parameters_.size() == 1);
+    user_assert(!this->is_array()) << "Cannot call the parameter() on Input<[]> " << name() << "; use an explicit subscript operator instead.";
```

Comment:
"the parameter()" does not parse for me. Could be either "the parameter() method" or "parameter()".

Reply:
Fixed